articles/python-projects-for-beginners/index.md

```diff
@@ -0,0 +1,196 @@
+
+In your [game.py](http://game.py)  file, import the random method and add the following:
+
+```python
+import random
+
+player_name= input("Enter your name: ")
+print(f"Welcome {player_name} to the rock, paper and scissor game\n")
+choice = ["rock", "paper", "scissors"]
+```
+Your terminal would display,
+
+![Input field](input.png)
+
+**Random** is a built-in module that generates random values within a particular specified range.
+
+To get the details from a user, we use the Python **input** function**.**
+
+*Next,* define a functioning game and set out the rules of the game.
+
+```python
+def game():
+ computer = random.choice(choice)
+ print("Rules of the game: \nrock vs scissors = scissor wins. \npaper vs scissors = scissors wins. \nrock vs paper = paper wins. \npaper vs rock = rock wins")
+ player = input("Your choice: ").lower()
+ print("Computer chose: " , computer)
+
+ if player == computer:
+    print("It is a draw, no winner!")
+ elif player == "rock" and computer == "paper":
+    print("Computer Wins!")
+ elif player == "rock" and computer == "scissors":
+    print("Computer wins!")
+ elif player == "scissors" and computer == "paper":
+    print("Computer wins!")
+ elif player == "scissors" and computer == " rock":
+    print("You win!")
+ elif player == "paper" and computer == "rock":
+    print("You win!")
+ elif player == "paper" and computer == "scissors":
+    print("You win!")
+ else:
+    print("Choose either paper, rock or scissor")
+
+game()
+```
+In your terminal, the game will be displayed like this.
+![Rock-paper-scissor](RPS.png)
+
+Notice, we use the *if-else* method in Python. They are very important in programming. With these methods, we can build programs that make decisions based on certain conditions. These are conditional statements. The ***if*** statement implies that if this is true or if it happens, then this should happen. While the ***elif*** and ***else*** statement simply implies that if the previous conditions don't happen or if it is not true, then this should happen.
+
+Notice, in this simple game, we have used a few methods in Python, such as *random, function, input, conditional statements.* I hope you properly understand how these concepts work in Python.
+
+Now let's move to something a bit more complex, building a *guessing game* with different levels where users will be able to choose the desired level and given limited chances to guess.
+
+To begin, create a Python file and add:
+
+```python
+import random
+
+user = input("Enter name: ")
+print(f"Welcome {user} to Guessing game101, have fun!")
+
+keep_playing = True
+while keep_playing:
+      print("\nThere are diferent levels: easy, medium and hard")
+      print("\nFor easy, you have 6 chances to guess a number between 1-10. \nFor meduim, you have 4 chances to guess a number between 1-20. \nFor hard, you have 4 chances to guess a number between 1-50")

```

Comment:
Instead of using newline character, use three print statements to improve readability